### PR TITLE
Use release branch of Nova workflow

### PR DESCRIPTION
**Pull request by spcyppt**
Differential Revision: D55947486

### CODE REVIEW

**Review by huydhn — APPROVED** [review]

**Comment by huydhn** [comment]
Wait a sec, I thought this is going to fbgemm release branch.  For the main branch, I think it's better to use the test-infra @main instead. And yes, we will add the option to fallback to `bdist_wheel`.  What I mean is that fbgemm release branch can use test-infra @release/2.3.  This is what we call a release-only change on PyTorch.

**Comment by spcyppt** [comment]
@huydhn It's also currently blocking our nightly releases, so I'm changing this in main to unblock nightly release. Once the fallback is added and merged, I will change back to main.

**Pull request closed without merging**
